api/types/desktop.go

```diff
@@ -160,3 +181,10 @@ func (d *WindowsDesktopV3) Origin() string {
 func (d *WindowsDesktopV3) SetOrigin(o string) {
 	d.Metadata.Labels[OriginLabel] = o
 }
+
+// MatchSearch goes through select field values and tries to
+// match against the list of search values.
+func (d *WindowsDesktopV3) MatchSearch(values []string) bool {
+	fieldVals := []string{d.GetName(), d.GetAddr(), fmt.Sprint(d.GetAllLabels())}
```

Comment:
Won't this also add `map` to the potential matched strings, due to the string representation of `map[string]string`?

Reply:
you're right (facepalm), I created a separate method to convert it to strings